debian/control

```diff
@@ -0,0 +1,18 @@
+Source: jackline
+Maintainer: Robur <debian@robur.coop>
+Build-Depends: debhelper
+Rules-Requires-Root: no
+Section: ocaml
+Priority: optional
+Homepage: https://github.com/hannesm/jackline/
+Vcs-Git: https://github.com/hannesm/jackline.git
+Vcs-Browser: https://github.com/hannesm/jackline/
+Standards-Version: 4.1.0
+
+Package: jackline
+Section: net
+Priority: optional
+Tag: implemented-in::ocaml, role::program
+Architecture: amd64
+Depends: ${shlibs:Depends}, ${misc:Depends}
```

Comment:
The `jackline` binary is dynamically linked against `libgmp.so` -- I'd expect this dependency to occur here?

Reply:
@hannesm the `${shlibs:Depends}` expands to `libgmp10`; since the `gmp` library changes package name with the version in Debian (to work around their rules that they can only have one version of each package installed).

```shell
dpkg --info jackline_0.0.1_amd64.deb 
 new Debian package, version 2.0.
 size 2438064 bytes: control archive=588 bytes.
     312 bytes,    11 lines      control              
     191 bytes,     3 lines      md5sums              
 Package: jackline
 Version: 0.0.1
 Architecture: amd64
 Maintainer: Robur <debian@robur.coop>
 Installed-Size: 15187
 Depends: libc6 (>= 2.29), libgmp10
 Section: net
 Priority: optional
 Homepage: https://github.com/hannesm/jackline/
 Description: XMPP client written in OCaml.
 Tag: implemented-in::ocaml, role::program
```